packages/discord.js/src/managers/GuildStickerManager.js

```diff
@@ -57,7 +57,7 @@ class GuildStickerManager extends CachedManager {
    *   .then(sticker => console.log(`Created new sticker with name ${sticker.name}!`))
    *   .catch(console.error);
    */
-  async create(file, name, tags, { description, reason } = {}) {
+  async create({ file, name, tags, description, reason } = {}) {
```

Comment:
Not longer optional:
```suggestion
  async create({ file, name, tags, description, reason }) {
```